src/main.rs

```diff
@@ -304,7 +295,8 @@ async fn run_test(test_args: TestArgs) -> anyhow::Result<i32> {
         repo_head_commit_epoch,
         tags: _,
         print_files: _,
-        dry_run,
+        dry_run: _,
+        no_quarantining,
         team,
         codeowners_path,
```

Comment:
nit: these don't need `_`, we can use `..` for partial destructuring
```suggestion
        no_quarantining,
        team,
        codeowners_path,
        ..
```